rpcs3/rpcs3qt/gui_settings.h

```diff
@@ -215,6 +215,7 @@ namespace gui
 
 	const gui_save m_currentStylesheet = gui_save(meta, "currentStylesheet", DefaultStylesheet);
 	const gui_save m_showDebugTab      = gui_save(meta, "showDebugTab",      false);
+	const gui_save m_attachCommandLine = gui_save(meta, "attachCommandLine", false);
```

Comment:
Actually, I think it's better to just use a cli command and do this in main.

Reply:
Better for who? I launch from visual studio.

Reply:
You can set cli args in visual studio

Reply:
Okay, I added both options in #15432.